src/MembershipManager.php

```diff
@@ -366,13 +380,6 @@ public function isMemberBlocked(EntityInterface $group, AccountInterface $user)
     return $this->isMember($group, $user, [OgMembershipInterface::STATE_BLOCKED]);
   }
 
-  /**
-   * {@inheritdoc}
-   */
-  public function reset() {
```

Comment:
IMO, This should have been deprecated instead of fully removed. I know we're not offering yet BC but still, given the fact that so many sites are already using OG...

Reply:
You're right, we could have deprecated this.
It broke tests in #181.